### PR TITLE
Separate contracts

### PR DESCRIPTION
**Pull request by Buckram123**
See ORC-152

### CODE REVIEW

File: packages/clone-testing/src/core.rs [hunk 1]
```diff
@@ -519,7 +519,7 @@ mod test {
     impl Uploadable for MockCw20 {
         fn wasm(_chain: &ChainInfoOwned) -> WasmPath {
             let path = PathBuf::from_str(env!("CARGO_MANIFEST_DIR")).unwrap();
-            let path = path.join("../../artifacts/cw20_base.wasm");
+            let path = path.join("../../../artifacts/cw20_base.wasm");
```
Comment (CyberHoward) [inline packages/clone-testing/src/core.rs:522]: Are you sure about this change?[#386](https://github.com/AbstractSDK/cw-orchestrator/issues/386)
Is this even ever used?

File: wasm_contracts.sh
Comment (CyberHoward) [inline wasm_contracts.sh:1]: let's move this to the contracts-ws dir

**Review by CyberHoward — APPROVED** [review]
Minor comments, LGTM!

File: packages/clone-testing/src/core.rs (hunk 1 above)
Comment (Buckram123) [reply packages/clone-testing/src/core.rs:522]: Nice catch, it's not being used, we don't even have this file. I'll put `unimplemented!()` inside

File: wasm_contracts.sh
Comment (Buckram123) [reply wasm_contracts.sh:1]: `artifacts/` still in the root of repo. Please check https://github.com/CosmWasm/optimizer?tab=readme-ov-file#contracts-excluded-from-workspace

**Comment by CyberHoward** [comment]
Could you fix the CI before we merge?

**Comment by CyberHoward** [comment]
@Buckram123 the links in the docs weren't update! 
image

**Pull request merged**